daemon/input.c

```diff
@@ -156,7 +159,7 @@ input_get_usb_input_event_file(uint16_t vendor_id, uint16_t product_id)
 }
 
 void
-pin_entry_timeout_cb(event_timer_t *timer, UNUSED void *data)
+pin_entry_timeout_cb(event_timer_t *t, void *data)
 {
```

Comment:
if global var is renamed to input_timer, no need to change the var name here

Reply:
Done